packages/init/templates/.eslintrc.hbs

```diff
@@ -1,39 +1,18 @@
 {
   "plugins": ["prettier"],
-  {{#if (eq type "node")}}
-  "extends": ["4catalyzer", "prettier", {{~#if typescript}}"4catalyzer-typescript"{{/if}}],
+  "extends": ["4catalyzer-react", "4catalyzer-jest", "prettier", "prettier-react", {{~#if typescript}}"4catalyzer-typescript"{{/if}}],
```

Comment:
uh, actually, I don't understand why jest is here. how does this apply only to the test folder and not to the rest of files?

Reply:
the config, like the typescript config is fully defined in an override for only test/** and *.test files

Reply:
ah I see. I was checking the `rules` file but not the index. makes sense